src/main/java/net/minestom/arena/game/mob/MobArena.java

```diff
@@ -365,8 +369,40 @@ private void onStageCleared() {
         bossBar.color(BossBar.Color.GREEN);
 
         group().playSound(Sound.sound(SoundEvent.UI_TOAST_CHALLENGE_COMPLETE, Sound.Source.MASTER, 0.5f, 1), Sound.Emitter.self());
+        arenaInstance.showTitle(Title.title(
+                Component.text("Stage Cleared", NamedTextColor.GREEN),
+                Component.text(ROUND_WAIT_TIME.getSeconds() + " seconds to prepare for next stage"),
+                Title.Times.times(Duration.ZERO, Duration.ofSeconds(2), Duration.ofMillis(300))
+        ));
         Messenger.info(group(), "Stage " + stage + " cleared! Talk to the NPC to continue to the next stage");
         new NextStageNPC().setInstance(arenaInstance, new Pos(0.5, HEIGHT, 0.5));
+
+        startForceContinueCountdown();
+    }
+
+    private void startForceContinueCountdown() {
```

Comment:
Could you also display the countdown in an action bar? Just an idea, not a must.